src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/LoggerMessageDefineAnalyzer.cs

```diff
@@ -0,0 +1,371 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeQuality.Analyzers;
+
+namespace Microsoft.Extensions.Logging.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
```

Comment:
Any reason this is a C#-only?

Reply:
this started off as a csharp-only analyzer (refer to dotnet/extensions link). I'd be open to consider making the analyzer language agnostic on a future PR, but for now it seems like the safest approach is to set it up as-is to start with.